src/components/testCaseList/CreateCodeCoverageNavTabs.tsx

```diff
@@ -155,7 +157,9 @@ export default function CreateCodeCoverageNavTabs(props: NavTabProps) {
                 _.isNil(measure?.groups) ||
                 measure?.groups.length === 0 ||
                 !executionContextReady ||
-                executing
+                executing ||
+                _.isNil(validTestCases) ||
+                _.isEmpty(validTestCases)
```

Comment:
I believe _.isEmpty() also performs a null check. Is there anything else we are expecting by adding _.nil()?

Reply:
isNil caused a test to fail, leaving isEmpty